dotCMS/src/main/java/com/dotcms/storage/StoragePersistenceProvider.java

```diff
@@ -0,0 +1,71 @@
+package com.dotcms.storage;
+
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.Logger;
+import java.io.File;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class StoragePersistenceProvider {
```

Comment:
Codacy found an issue: [A class which only has private constructors should be final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6220272)